crypto/fipsmodule/modes/xts_test.cc

```diff
@@ -583,6 +597,254 @@ static const XTSTestCase kXTSTestCases[] = {
+        "656d1bdc65fdc3",
+        "4100e99913513a39bd379f672ac3fa13e6bab8412d3373531a801f70ea9b5e25"
+        "603229eac8b0af2d0619c1859bb980ee08e1d43e4c3fa206ff0928eaac063978"
+        "0e1e25dea0a6e4e25c8d27793425e740c1bac1bcf98986125372c12198819028"
+        "40b7d612adc275de1a1bbeea216458e1010cf7019a949a4a3f016edbd0b09226"
+        "9295e7924a2d9267c1babb021b0f0e207de84460c55cdb4abc6b2dae0960d9af"
+        "1a95f1dfcc83ec625bd7c42b1e17d219b15d5cc0e3e69063889f1ab18008c309"
+        "14b549815822b844b279324cd7cf8f596445b1abbd87ee97c13242bd11958c0d"
+        "45986cd54247c307224568d557d42188f39f1ce6338cb3aa3c18501095ab5316"
+        "db18cb23698aa57fb12728a747b2f64878bc23e260cae1d597214a817d8b0229"
+        "fa449b6e55aad6260b230049b4a7708a9d72d06208033b40305bacd253ec4c5b"
+        "e85d813d4adbd69083bcb7bcda1667d30e0315753e1801c165b7f8a8be6a0717"
+        "c6d5e3ce942c57"
+    },
+    // len = 370 bytes = 23 blocks + 2 bytes
+    {
+        "f4269bca3fb01715a422ab9f9dedb7b4218e8a004af9216e687cc024d55fc239"
+        "71e2b043f12f710d59b63011ba02b0acdabc59c9b0610ec590131b5d128d14e4",
+        "eec7a07920a0426a503bcb724a3a37b2",
+        "cab3c58bceeae8a4fc4831be5c6aa24a3143c352e306bc344188a68bc2dd3d8c"
+        "9103175fed0003e94834a7a49e4aeed08db22270b8dea4f9664a852828c2b5b9"
+        "c5cc18b2cc1c9b145043b8ef8da7bf1a59e18a11bf2f0a26798f4ea152035a17"
+        "d073ca9c8f65b1dfa869ce35108d4f696eda7a2e0985548214a22466a67e7e76"
+        "f1481280adc360562d2e8b3dbcdba72ab52158bea6ac40bb4f6421f5e39f6bd4"
+        "e77d559541b5eb6ee376ab9f5152ca067422c41acf05d51e69f7134c967e217e"
+        "fb76133c2bfeaa0e7456aec6a878cc1c9a913769960c87ff039a4c9a186d1814"
+        "e32b500e29fb1c9d51ca63f9423016ddc14d465759ce565d68a2f7810f0f95f2"
+        "3ae50063e01d0031e7642b2a944107558e4dace71b024484a53b05b44a9aa784"
+        "7fa7e760c4e891ac4cbcd6e0fddd358b2ae17346e3b7ca88f3cf3d3d69e4c2e8"
+        "8ba9485091dafcdd96d2bd94aff21fd9d3921fb74ae93f3db87c7a21603c0aec"
+        "e6523c772c3855c30a1257b9057692d809b2",
+        "74468138b84656e4292b2855019566852232e2126ad7e062b9a9320027e4067a"
+        "a19d70f4a338bdd30c6e08a3d42e0c4b5d75d4bda6078df47469fec4dac80b49"
+        "c80e0076dd1bf773919d8699455bbc905c1ba7d8e968d15ad0525ad31d080335"
+        "15c4e85068fe0eede28ecd3fd434e0f8db6db3d99c0c9120d060b21490e15b4c"
+        "23a5db14c06a02d5677f64f2e861ad7333180417d6eae190a9f5fcfbb629795a"
+        "c9b8d5792767ec63f7d5146aaf3294c8924c951a0b95221dc821d8bc428a7670"
+        "cf4c900f7d844bd0b48dfbf70f18718fe404bdd476fbf366a01a1bf69d3178b1"
+        "0981388a1d51c05e97cc5ae6061ea57736b22989e772fa413eb89e25357827f5"
+        "89d41dfbf6dc6cfeb063a2cd3cf2da200b99b4a856a8aff15f27769f9d20ceee"
+        "12bf2b09391d9c75d27f69417a90b198240df02fb4abb351fbd0b753f50c9349"
+        "59ad6b5d74d0677c792a89c3c20f22285ae56aaea231b27fc0be998320cc395c"
+        "42d9a1b72c80603465ca9bdb3960f0f7a534"
+    },
+    // len = 399 bytes = 24 blocks + 15 bytes
```

Comment:
I think it would be good to add test vectors for:
- 32 blocks
- one 32 blocks + x bytes
- one 32+x blocks, for example 34

Reply:
will update the test.